dev-util/SPIRV-Headers/SPIRV-Headers-1.3.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils vcs-snapshot
+
+SNAPSHOT_COMMIT="02ffc719aa9f9c1dce5ce05743fb1afe6cbf17ea"
+SRC_URI="https://github.com/KhronosGroup/SPIRV-Headers/archive/${SNAPSHOT_COMMIT}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64"
+
+DESCRIPTION="Machine-readable files for the SPIR-V Registry"
+HOMEPAGE="https://www.khronos.org/registry/spir-v/"
+
+LICENSE="Khronos"
+SLOT="0"
+
+src_install() {
+	cd "${BUILD_DIR}"
+	make install-headers || die
```

Comment:
This is infinately wrong, and I don't understand how this ebuild could work in the first place:
1. You don't reference `${D}`, so how the build system knows where to install?
2. Never ever call `make` directly, use `emake` instead.
3. Never call `emake` in cmake-utils ebuilds, it's going to break stuff for people using Ninja. Use `cmake-utils_src_make` instead. Or force Makefile generator if it can't work with Ninja.

Reply:
this section has been removed